src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,14 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers
+                .stream()
+                .flatMap(string -> Arrays.stream(string.split(",")))
+                .mapToInt(Integer::parseInt)
+                .boxed()
+                .filter(number -> number % 2 == 0)
+                .min(Comparator.comparingInt(Integer::intValue))
+                .orElseThrow(() -> new RuntimeException("Can't get min value from list"));
```

Comment:
```suggestion
        return numbers
                .stream()
                .flatMap(string -> Arrays.stream(string.split(",")))
                .mapToInt(Integer::parseInt)
                .filter(number -> number % 2 == 0)
                .min()
                .orElseThrow(() -> new RuntimeException("Can't get min value from list"));
```
This is an unnecessary complication